server/sync.go

```diff
@@ -54,6 +56,54 @@ func transactionCommitInformer() chan int {
 		transactionCommited = make(chan int, 1)
 	}
 	return transactionCommited
+
+}
+
+var keyUpdated map[string]*list.List
+var subsListMutex sync.Mutex
+
+func keyUpdateSubscribers(key string) *list.List {
+	if keyUpdated == nil {
+		log.Critical("Creating key update subs")
+		keyUpdated = make(map[string]*list.List)
+	}
+	if _, ok := keyUpdated[key]; !ok {
+		log.Critical("Creating update sub list for new key %s", key)
+		keyUpdated[key] = list.New()
+	}
+	return keyUpdated[key]
+}
+
+func notifyKeyUpdateSubscribers(fullKey string) {
```

Comment:
This will work only for single node. It should be distributed to other HA nodes as well.